backend/src/main/java/ai/verta/modeldb/App.java

```diff
@@ -472,7 +472,9 @@ private static void initializeBackendServices(
     LOGGER.trace("ExperimentRun serviceImpl initialized");
     wrapService(serverBuilder, new JobServiceImpl(authService, jobDAO));
     LOGGER.trace("Job serviceImpl initialized");
-    wrapService(serverBuilder, new CommentServiceImpl(authService, commentDAO));
+    wrapService(
```

Comment:
with shadow delete direct access to comments gave access to comments, to put the call behind auth needed this change